File: MODUL 5 Praktikum Pemrograman/PRAKTIKUM 5 C/PRAK502-2410817120021-Aulia-Az-Zahra.c
#include <stdio.h>
#include <stdlib.h>
int hitung(int nilai1, int nilai2)
{
    //Lengkapi Function ini
    return abs(nilai1 - nilai2);
}
int mutlak(int angka)
{
    //Lengkapi Function ini
    return abs(angka);
}
int main()
{
    int a,b,c,d;
    scanf("%d", &a);
    scanf("%d", &c);
    scanf("%d", &b);
    scanf("%d", &d);

    int Hasil = hitung(a,b)+hitung(c,d);
    printf("%d", mutlak(Hasil));

    return 0;
    
}

File: MODUL 5 Praktikum Pemrograman/PRAKTIKUM 5 PY/PRAK502-2410817120021-Aulia-Az-Zahra.py
def hitung(nilai1, nilai2):
    #Lengkapi Function ini
    return abs(nilai1 - nilai2)

def mutlak(angka):
    #Lengkapi Function ini
    return abs(angka)

a,c,b,d = map(int, input().split())
Hasil = hitung(a, b)+hitung(c,d)
print(mutlak(Hasil))